### PR TITLE
135 project 8 spec

### PR DESCRIPTION
**Pull request by dylanvu**

### CODE REVIEW

File: project_8.html [hunk 1]
```diff
@@ -0,0 +1,793 @@
...
+				<h3 class="mb-3" id="pj-heading-pcb-design-overview">PCB Design Overview</h3>
+
+				<p class="mb-5">
+					When it is time to design your PCB, there are two major components of the operation: (1) Schematic
+					capture - converting
+					your prototyped design and circuit drawings into a digital schematic. (2) PCB Layout Design -
+					placing components and
+					routing conductive tracks in a virtual layout of your PCB. You can learn exactly how this process
+					works in the KiCad EDA
+					software by viewing our PCB Design with KiCad workshop material the lecture material <a
```
Comment (bcbielecki) [inline project_8.html:162]: Delete the typo "the lecture material"

File: project_8.html [hunk 2]
```diff
@@ -0,0 +1,793 @@
...
+					</li>
+				</ol>
+
+				<h2 class="mb-3 fw-bold" id="pj-heading-parts">Parts</h2>
+				<p class="mb-3">
+					This is not a physical parts list. The followig parts will be used in the PCB EDA application.
+				</p>
+
+				<!-- ======= Part Table ======= -->
+				<table class="table pj-part-table mb-5">
```
Comment (bcbielecki) [inline project_8.html:231]: Someone's table isn't tabling. The right outline/border is missing. I would recommend a little debugging. If all else fails, remove "pj-part-table" from the class attribute.

File: project_8.html [hunk 3]
```diff
@@ -0,0 +1,793 @@
...
+				</div>
+				<p class="mb-3">
+					<strong>Make sure to save your progress frequently.</strong> Nothing is more infuriating than trying
+					to recover lost data and lost time.
+				</p>
+				<p class="mb-3">
+					<strong>You should also do yourself the favor of using a mouse.</strong> Suffice to say, KiCad is
+					unkind to touchpad users (and if you are using a pointing stick, something is wrong with you.)
+				</p>
+				<p class="mb-3">
```
Comment (bcbielecki) [inline project_8.html:375]: Change to mb-5. Same applies to all other headers. The preceding block element should have a bottom margin of 5rem.

**Review by bcbielecki — CHANGES_REQUESTED** [review]
I have a couple comments for you to resolve. Nothing major except you will need to implement the Requirements section, which you will now find in the project doc.

File: project_8.html [hunk 4]
```diff
@@ -0,0 +1,793 @@
...
+							<td>
+								Package_DIP
+							</td>
+							<td>
+								DIP-8_W7.62mm
+							</td>
+						</tr>
+
+						<tr>
+							<td>x1</td>
```
Comment (bcbielecki) [inline project_8.html:275]: This should be x2. This was my bad.

**Review by bcbielecki — CHANGES_REQUESTED** [review]
One more comment...

**Review by bcbielecki — CHANGES_REQUESTED** [review]
Make sure to add the project to the navbar!!

File: project_8.html (hunk 1 above)
Comment (dylanvu) [reply project_8.html:162]: done

File: project_8.html (hunk 2 above)
Comment (dylanvu) [reply project_8.html:231]: the issue is because the footprint description is too long. I'll try to add some kind of text-wrapping

Comment (dylanvu) [reply project_8.html:231]: fix rn: just reduce font size. However, table is broken for mobile.

File: project_8.html [hunk 5]
```diff
@@ -0,0 +1,793 @@
...
+								<a class="nav-link" href="#pj-heading-printed-circuit-boards">Printed Circuit Boards
+									(PCB)</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-pcb-design-overview">PCB Design Overview</a>
+							</li>
+						</ul>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="#pj-heading-requirements">Requirements</a>
```
[the remark this answers is not in the document]
Comment (dylanvu) [reply project_8.html:70]: done

File: project_8.html (hunk 4 above)
Comment (dylanvu) [reply project_8.html:275]: done

File: project_8.html (hunk 3 above)
Comment (dylanvu) [reply project_8.html:375]: this is a little unclear to me. Do you mean that all <p> should have an mb-5?

File: project_8.html (hunk 2 above)
Comment (dylanvu) [reply project_8.html]: fixed. Added table responsive class: https://getbootstrap.com/docs/4.0/content/tables/#always-responsive

**Review by bcbielecki — APPROVED** [review]

**Pull request merged**